### DISCUSSION TITLE
Can users be tagged?

### DISCUSSION THREAD

**Discussion by roke-julian-lockhart** [getfider/fider#1224] [Q&A] [answered]
I've tried to tag a user at [`feedback.fider.io/posts/323/remove-upvotes-and-comments-from-blocked-users#comment-46456`](https://feedback.fider.io/posts/323/remove-upvotes-and-comments-from-blocked-users#comment-46456) using `@`, but it doesn't linkify.

**Comment by mattwoberts** [comment, marked as answer]
Hey @RokeJulianLockhart 

No you can't ... yet.

I'm working on this though right now, so it's the next feature that will be released :)

Here's a sneak preview ...

<img width="868" alt="image" src="https://github.com/user-attachments/assets/10ff3c5f-1341-45e9-9f9d-06905f02e79c">


Matt

**Reply by roke-julian-lockhart** [reply]
@mattwoberts, thank you! Is an issue tracking it?

**Reply by mattwoberts** [reply]
🤦‍♂️ There should be, but well.. I was just messing about with the idea one day and it grew and grew... I will make one though, thanks for reminding me
